README.md

```diff
@@ -4,28 +4,70 @@ Lexical is a next-generation language server for the Elixir programming language
 
 ## Features
 
-  * Code Completion
+  * Context aware code completion
   * As-you-type compilation
   * Advanced error highlighting
   * Code actions
   * Code Formatting
   * Completely isolated build environment
-  
+
+## What makes Lexical different?
+There are a couple things that lexical does differently than other language servers. Let's look at what separates it from
+the pack.
+
+#### Architecture
+
+When lexical starts, it boots an erlang virtual machine that runs the language server and its code. It then boots a
+separate virtual machine that runs your project code and connects the two via distribution. This provides the following benefits:
+
+  * None of lexical's dependencies will conflict with your project. This means that lexical can make use of dependencies to make developing in it easier without having to "vendor" them. It also means that you can use lexical to work on your project, **even if lexical depends on your project**.
+  * Your project can depend on a different version of elixir and erlang than lexical itself. This means that lexical can make use of the latest versions of elixir and erlang while still supporting projects that run on older versions.
+  * The build environment for your project is only aware of your project, which enables as-you-type compilation and error reporting.
+  * In the future, there is a possibility of having the lexical vm instance control multiple projects
+
+#### Ease of contribution
+
+Lexical has been designed to be easy to contribute to and work on. It features:
+
+  * A consistent data model that represents the Language Server Protocol. No more working with maps and guessing what the keys of a particular message are.
```

Comment:
I know you've voiced your frustration with this aspect of elixir_sense and elixir_ls, and I understand that frustration having contributed to elixir_ls myself, but this feels unnecessary. I'm all in for talking about the data model as an advantage, but I don't think we need to take directed shots at other projects in the space, even if we're not naming them explicitly.

Reply:
Maybe `A consistent data model that represents the Language Server Protocol.` is enough.

Reply:
Right, that was not my intention, I was just providing contrast. Removed.